quickstarts/jboss-eap8-with-helm.yaml

```
@@ -0,0 +1,215 @@
+
+          - The first build configuration **eap8-build-artifacts** compiles and packages the Jakarta EE application, and creates a JBoss EAP server.
+            The application is run on this JBoss EAP server.
+
+            The build may take a few minutes to complete. The build state is indicated by a relevant message such as **Pending**, **Running**, and **Complete**.
+
+            When the build is complete, a checkmark and the following message is displayed: **Build #1 was complete**.
+
+            When the first build is complete, the second build starts.
+
+          - The second build configuration **eap8** puts the Jakarta EE deployment and the JBoss EAP server in a runtime image that contains only what is required to run the  application.
+
+            When the second build is complete, a checkmark and the following message are displayed: **Build #2 was complete**.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: >-
+          The two builds for **eap8-build-artifacts** and **eap8** may take a few minutes to complete.
+
+          Verify the builds are complete:
+
+          - The message **Build #1 was complete** is displayed for the **eap8-build-artifacts** build configuration. Did this message appear?
+
+          - The message **Build #2 was complete** is displayed for the **eap8** build configuration. Did this message appear?
+      summary:
+        failed: Try the steps again.
+        success: Your build is complete.
+      title: View the Build status
+    - description: >-
+        To view the pod status:
+
+        1. In the navigation menu, click [Topology]{{highlight qs-nav-topology}}.
+
+        1. In the **Topology** view, click **D eap8**.  
+           A side panel opens with detailed information about the application.
+
+        1. In the **Details** tab, the pod status is available in a tooltip by hovering over the pod.
+
+            - Inside the pod circle, it displays the number of pod.
+            - The color of the pod circle indicates the pod status:
+              Light blue = **Pending**, Blue = **Not Ready**, Dark blue = **Running**
+
+        **Note:** In the **Topology** view, the dark outer circle indicates the pod status.
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again.
+        instructions: |-
+          Verify you see the pod status:
+          
+          - Does the text inside the pod cirle display **1 Pod**?
+
+          - When you hover of the pod circle, does it display **1 Running**?
+
+      summary:
+        failed: Try the steps again.
+        success: Your deployment has one running pod.
+      title: View the Pod status
+    - description: >-
+
+        To view the JBoss EAP application:
+
+        1. In the **Topology** view, click the external link icon in the top right-hand corner to open the URL and run the application in a separate browser window.
```

```suggestion
        1. In the **Topology** view, click the external link icon in the top right corner to open the URL and run the application in a separate browser window.
```

I will say that in docs we avoid directionals for accessibility reasons, but I would defiantly make it "right" instead of "right-hand" at least.

---

fixed, thanks